docker/docker-compose.ci.yml

```diff
@@ -147,6 +147,8 @@ services:
       - /app/apps/client/node_modules/
       - ../packages/shared:/app/packages/shared
       - /app/packages/shared/node_modules/
+      - ../packages/test-utils:/app/packages/test-utils
+      - /app/packages/test-utils/node_modules/
```

Comment:
`[vite] Internal server error: Failed to resolve import "test-utils" from "cypress/components/specs/repo-form.ct.js". Does the file exist?`

Reply:
oui,le répertoire est présent [dso-console](https://github.com/dnum-mi/dso-console/tree/feat/gitlab-ci-generator)/[packages](https://github.com/dnum-mi/dso-console/tree/feat/gitlab-ci-generator/packages)/test-utils/

As-tu reconstruit tes containers?